sql/parser/sql.y

```diff
@@ -2774,14 +2774,6 @@ func_expr_windowless:
 // Special expressions that are considered to be functions.
 func_expr_common_subexpr:
   COLLATION FOR '(' a_expr ')' { unimplemented() }
-| CURRENT_DATE
```

Comment:
We decided to not go the standards route because it felt out of place and
is not part of the modern approach. We can always add in the standards
style later.

On Thu, Sep 24, 2015, 3:16 PM Peter Mattis <notifications@github.com> wrote:

> In sql/parser/sql.y
> <https://github.com/cockroachdb/cockroach/pull/2630#discussion_r40360285>:
>
> > @@ -2774,14 +2774,6 @@ func_expr_windowless:
> >  // Special expressions that are considered to be functions.
> >  func_expr_common_subexpr:
> >    COLLATION FOR '(' a_expr ')' { unimplemented() }
> > -| CURRENT_DATE
>
> I think you should revert this. The variable form of
> CURRENT_{DATE,TIMESTAMP} is part of the SQL standard.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cockroachdb/cockroach/pull/2630/files#r40360285>.
>